datastore_entity/src/lib.rs

```diff
@@ -43,23 +47,15 @@ impl Display for DatastoreEntity {
     }
 }
 
-pub trait DatastoreFetch<T> {
-    fn get_by_id<A>(id: i64, token: A, project_name: &String) -> Result<T, String>
-    // TODO - error format
-    where
-        A: ::google_api_auth::GetAccessToken + 'static;
-}
-
-pub fn get_one_by_id<A>(
+pub fn get_one_by_id<T>(
     id: i64,
     kind: String,
-    token: A,
-    project_name: &String,
+    connection: &T
```

Comment:
there is a much easier way with sugar!

`connection: &impl DatastoreConnection`

and then drop the where clause and the generic `T`

Reply:
👍

Reply:
fixed in https://github.com/andreaspsson/datastorers/pull/16